nemo_text_processing/inverse_text_normalization/ja/verbalizers/post_processing.py

```diff
@@ -16,10 +16,18 @@
 import os
 
 import pynini
-
-from nemo_text_processing.text_normalization.en.graph_utils import (
+from pynini.lib import pynutil
+
+from nemo_text_processing.inverse_text_normalization.ja.graph_utils import (
+    NEMO_CHAR,
+    NEMO_DIGIT,
+    NEMO_NARROW_NON_BREAK_SPACE,
```

Comment:
let's apply the changes suggested below by codeql

Reply:
Fixed imports